src/sentry/migrations/0161_add_saved_search_sort.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 1.11.29 on 2021-02-10 04:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0160_create_projectcodeowners_table"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="savedsearch",
+            name="sort",
+            field=models.CharField(
+                choices=[
+                    ("date", "Last Seen"),
+                    ("priority", "Priority"),
+                    ("new", "First Seen"),
+                    ("freq", "Frequency"),
+                    ("user", "Users"),
+                ],
+                default="date",
+                max_length=16,
+            ),
+        ),
+    ]
```

Comment:
So that we can set the default in Django without causing an sql default to be set, let's do:
```
    operations = [
        migrations.SeparateDatabaseAndState(
            database_operations=[
                migrations.AddField(
                    model_name="savedsearch",
                    name="sort",
                    field=models.CharField(
                        choices=[
                            ("date", "Last Seen"),
                            ("priority", "Priority"),
                            ("new", "First Seen"),
                            ("freq", "Frequency"),
                            ("user", "Users"),
                        ],
                        null=True,
                        max_length=16,
                    ),
                ),
            ],
            state_operations=[
                migrations.AddField(
                    model_name="savedsearch",
                    name="sort",
                    field=models.CharField(
                        choices=[
                            ("date", "Last Seen"),
                            ("priority", "Priority"),
                            ("new", "First Seen"),
                            ("freq", "Frequency"),
                            ("user", "Users"),
                        ],
                        default="date",
                        null=True,
                        max_length=16,
                    ),
                ),
            ],
        ),

    ]
```